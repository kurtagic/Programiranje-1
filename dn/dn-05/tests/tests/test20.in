53
3 100
11 100
0 100
7 100
11 100
3 100
7 100
11 100
11 100
14 100
0 100
11 100
1 100
8 100
8 100
3 100
1 100
7 100
14 100
7 100
8 100
0 100
8 100
14 100
11 100
0 100
8 100
14 100
0 100
0 100
0 100
1 100
8 100
7 100
8 100
2 100
0 100
14 100
0 100
8 100
0 100
2 100
3 100
3 100
14 100
3 100
8 100
11 100
8 100
1 100
11 100
2 100
7 100
