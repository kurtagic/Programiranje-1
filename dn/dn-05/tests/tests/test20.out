100: 111
101: 109
102: 100
103: 84
